compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeSceneMediator.uikit.kt

```diff
@@ -162,12 +199,61 @@ internal class ComposeSceneMediator(
         )
     }
 
-    private val delegate: SkikoUIViewDelegate by lazy {
-        SkikoUIViewDelegateImpl(
-            { scene },
-            interopContext,
-            densityProvider,
-        )
+    private val touchesDelegate: TouchesDelegate by lazy {
```

Comment:
What do you think about renaming `TouchesDelegate` to `InteractionUIViewDelegate` or make it inner `Delegate` interface for `InteractionUIView`, it doesn't seem to be a general case interface whose functions are invoked by something else.

Reply:
Good suggestion

Reply:
Done